marklogic-client-api/src/test/resources/dataservices/bulkOutputCallerNext.sjs

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const state  = fn.head(xdmp.fromJSON(endpointState));
+const work = fn.head(xdmp.fromJSON(workUnit));
+const limit = fn.head(xdmp.fromJSON(workUnit));
```

Comment:
Is there a reason to get the work unit twice?  Would it work just as well with work.max instead of limit.max?

Also, would "limit" indicate the purpose of the property better than "max"?